lib/livebook_web/live/session_live.ex

```diff
@@ -228,93 +228,65 @@ defmodule LivebookWeb.SessionLive do
-                  <%= hub.emoji %>
-                  <span class="ml-2"><%= hub.name %></span>
-                </button>
-              </.menu_item>
-            </.menu>
-
-            <h1
-              class="grow p-1 -ml-1 text-3xl font-semibold text-gray-800 border border-transparent rounded-lg whitespace-pre-wrap"
-              tabindex="0"
-              id="notebook-heading"
-              data-el-heading
-              spellcheck="false"
-              phx-no-format
-            ><%= @data_view.notebook_name %></h1>
-
-            <.menu id="session-menu">
-              <:toggle>
-                <button class="icon-button" aria-label="open notebook menu">
-                  <.remix_icon icon="more-2-fill" class="text-xl" />
-                </button>
-              </:toggle>
-              <.menu_item>
-                <.link patch={~p"/sessions/#{@session.id}/export/livemd"} role="menuitem">
-                  <.remix_icon icon="download-2-line" />
-                  <span>Export</span>
-                </.link>
-              </.menu_item>
-              <.menu_item>
-                <button role="menuitem" phx-click="erase_outputs">
-                  <.remix_icon icon="eraser-fill" />
-                  <span>Erase outputs</span>
-                </button>
-              </.menu_item>
-              <.menu_item>
-                <button role="menuitem" phx-click="fork_session">
-                  <.remix_icon icon="git-branch-line" />
-                  <span>Fork</span>
-                </button>
-              </.menu_item>
-              <.menu_item>
-                <a role="menuitem" href={live_dashboard_process_path(@session.pid)} target="_blank">
-                  <.remix_icon icon="dashboard-2-line" />
-                  <span>See on Dashboard</span>
-                </a>
-              </.menu_item>
-              <.menu_item variant={:danger}>
-                <.link navigate={~p"/home/sessions/#{@session.id}/close"} role="menuitem">
-                  <.remix_icon icon="close-circle-line" />
-                  <span>Close</span>
-                </.link>
-              </.menu_item>
-            </.menu>
+                    </.menu_item>
+                  </.menu>
+                </div>
+
+                <div class="flex md:hidden">
+                  <.headline_menu session={@session} />
+                </div>
+              </div>
+
+              <div class="flex grow">
+                <h1
+                  class="p-1 text-3xl font-semibold text-gray-800 border border-transparent rounded-lg whitespace-pre-wrap"
+                  tabindex="0"
+                  id="notebook-heading"
+                  data-el-heading
+                  spellcheck="false"
+                  phx-no-format
+                ><%= @data_view.notebook_name %></h1>
+              </div>
+            </div>
+
+            <div class="hidden md:flex">
+              <.headline_menu session={@session} />
+            </div>
```

Comment:
I think we can structure the elements like this:

```
[[Name]             [[Hub][Menu]]
```

On desktop we position [Hub] absolutely and on mobile we display the outer element as column. Probably simpler, but I may be missing something :)